nl2sql/llms/vertexai.py

```diff
@@ -79,12 +86,18 @@ def model(
     """
     Return an Instance of Vertex AI LLM
     """
-    return ExtendedVertexAI(
-        model_name=model_name,
-        max_tokens=ExtendedVertexAI.input_token_limits.get(model_name, max_output_tokens),
+    if not model_name in ExtendedVertexAI.model_token_limits:
+        raise NotImplementedError
```

Comment:
If the user provides a model name that we don't know about, we should still try to create the LLM object.